lokalise.rb

```diff
@@ -8,46 +8,46 @@ def self.run(params)
         project_identifier = params[:project_identifier]
         destination = params[:destination]
         clean_destination = params[:clean_destination]
-        include_comments = params[:include_comments] ? 1 : 0
-        use_original = params[:use_original] ? 1 : 0
-
-        request_data = {
-          api_token: token,
-          id: project_identifier,
-          type: "strings",
-          use_original: use_original,
+        include_comments = params[:include_comments]
+        original_filenames = params[:use_original]
+
+        body = {
+          format: "ios_sdk",
+          original_filenames: original_filenames,
           bundle_filename: "Localization.zip",
           bundle_structure: "%LANG_ISO%.lproj/Localizable.%FORMAT%",
-          ota_plugin_bundle: 0,
-          export_empty: "base",
-          include_comments: include_comments
+          export_empty_as: "base",
+          export_sort: "first_added",
+          include_comments: include_comments,
+          replace_breaks: false
         }
 
-        languages = params[:languages]
-        if languages.kind_of? Array then
-          request_data["langs"] = languages.to_json
+        filter_langs = params[:languages]
+        if filter_langs.kind_of? Array then
+          body["filter_langs"] = filter_langs.to_json
```

Comment:
`.to_json` Should be removed. Otherwise, an error occurs
```suggestion
          body["filter_langs"] = filter_langs
```

Reply:
You're right. Thanks!